### ISSUE TITLE
is the Build rich interactions ready to use?

### ISSUE THREAD

**Issue by sunsgs** [snipsco/snips-platform-documentation#26]
I would like to know if we can already implement build rich interactions. Thanks

**Comment by tschmidty69** [comment]
Definitely. Requires some creativity and custom handler code like any good intent but I have things like:

Would you like me to close the garage door?
Yes
OK, closing the garage door

I also have it detecting motion and

Good Morning, Sir.
Good Morning, Jarvis
The weather is blah, blah, would you like to hear the latest news
Yes
OK, the top 3 headlines today are blah, blah
